src/content/1.7/contribute/i-need-help.md

```diff
@@ -0,0 +1,25 @@
+---
+title: I need help
```

Comment:
I dont know if this page should go into "Contribute" or "FAQ".

In "Contribute" there is "Reporting issues" so that is why I initially put it there.